tests/sdk/test_file_uploads.py

```
@@ -0,0 +1,254 @@
+import json
+
+import pytest
+
+from sdks.python.src.honcho.async_client.client import AsyncHoncho
+from sdks.python.src.honcho.client import Honcho
+
+
+@pytest.mark.asyncio
+async def test_session_upload_file(
```

Ruff (0.12.2)

Missing return type annotation for public function `test_session_upload_file`

Add return type annotation: `None`

(ANN201)